core/src/main/resources/hudson/model/AbstractBuild/index.jelly

```diff
@@ -25,6 +25,13 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
+  <j:if test="${!h.isAnonymous()}">
+    <j:invokeStatic var="userHasDisplayTimeZone" className="hudson.timezone.DisplayTimezoneProperty" method="isCurrentUserUseDisplayTimezone" />
+    <j:if test="${userHasDisplayTimeZone}">
+      <j:invokeStatic var="userDisplayTimeZone" className="hudson.timezone.DisplayTimezoneProperty" method="getCurrentUserDisplayTimezone" />
+      <i:setTimeZone value="${userDisplayTimeZone}"/>
+    </j:if>
+  </j:if>
```

Comment:
Can this be in `layout.jelly` so we do not have to put it in all the views (or even worse, present different dates in different timezones causing confusion)?